tests/test_messages.py

```diff
@@ -1,12 +1,18 @@
+from enum import Enum
+from ipaddress import IPv4Address, IPv6Address
+from pathlib import Path
 from typing import Annotated, List, Literal, Union
 
-from pydantic import BaseModel
-from pydantic.error_wrappers import ValidationError
+import pytest
+from pydantic import BaseModel, field_validator
 from pydantic.fields import Field
 from pydantic.networks import AnyHttpUrl
-import pytest
+from pydantic_core import ValidationError
+
+from fastapi_validation_i18n._helpers import translate_errors
+from fastapi_validation_i18n.translator import Translator
 
-from fastapi_validation_i18n.handler import Helper
+translator = Translator('zh-TW', locale_path=str(Path.cwd()) + '/tests/locale')
```

Comment:
**Suggestion:** Use f-strings for string concatenation for better readability and performance.
```suggestion
translator = Translator('zh-TW', locale_path=f"{Path.cwd()}/tests/locale")
```